README.md

```diff
@@ -1,2 +1,69 @@
-# bookstore
-Bookstore is an SPA made for ethusiastic readers. it is a content management app that helps you track your progress on your reading journey. It is built using React.js
+# Bookstore
+
+![Microverse](https://img.shields.io/badge/Microverse-blueviolet)
+
+> **Bookstore** is an SPA made for ethusiastic readers. it is a content management app that helps you track your progress on your reading journey.
+> It is built using React.js
+
+## Built With
+
+- React (CRA)
+
+## Milestones
+
+- [x] Milestone 1: **Initialize project with components (React only).**
+- [x] Milestone 2: **Add reducers and actions.**
+- [x] Milestone 3: **Use Redux in React components.**
+- [x] Milestone 4: **Connect to API.**
+- [x] Milestone 5: **Styling**
+
```

Comment:
- Please, kindly remove this section as they're not relevant to the project and is not a section in the readme file.